### PR TITLE
updating vb template icon

### PR DESCRIPTION
**Pull request by jmarolf**
**Customer scenario**

Vb class library templates appear with C# icon

**Bugs this fixes:** 

**[Bug 429537](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/429537)**

**Workarounds, if any**

None

**Risk**

None

**How was the bug found?**

ad hoc testing

### CODE REVIEW

**Comment by jmarolf** [comment]
@srivatsn @natidea @dotnet/project-system please review

File: src/Templates/Microsoft.NetCore.VB.ProjectTemplates/ClassLibrary.vstemplate [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
   <TemplateData>
     <Name Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="9"/>
-    <Description Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="10"/>
-    <Icon Package="{FAE04EC1-301F-11d3-BF4B-00C04F79EFBC}" ID="4547" />
+    <Description Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="6"/>
+    <Icon Package="{164B10B9-B200-11D0-8C61-00A0C91E29D5}" ID="4500" />
```
Comment (srivatsn) [inline src/Templates/Microsoft.NetCore.VB.ProjectTemplates/ClassLibrary.vstemplate]: Can you update the unit test projects as well while you are here?

**Comment by srivatsn** [comment]
@ManishJayaswal @Pilchie for approval.

**Comment by natidea** [comment]
Approved to merge via [429537](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/429537)

**Pull request merged**